packages/modular-scripts/src/utils/filterDependencies.ts

```diff
@@ -16,16 +17,29 @@ export function filterDependencies(
       external: {},
     };
   }
+
+  // By default, nothing is in blocklist
   const externalBlockList =
     process.env.EXTERNAL_BLOCK_LIST && !isApp
       ? process.env.EXTERNAL_BLOCK_LIST.split(',')
       : [];
+
+  // By default, everything in allow list
+  const externalAllowList =
+    process.env.EXTERNAL_ALLOW_LIST && !isApp
```

Comment:
We should add documentation for this before we get too deep adding APIs and can't remember what's there and not.